ruby_on_rails/advanced_forms_and_activerecord/active_record_queries.md

```diff
@@ -204,10 +205,92 @@ You might be thinking, Why use a scope when you can write a class method to do t
   ...
 ```
 
-See the Additional Resources section for links to some posts that dig a bit deeper into the use cases for these two.
+See the [Additional Resources section](#additional-resources) for links to some posts that dig a bit deeper into the use cases for these two.
 
 How much do you need to understand or care about scopes? In the early going, you probably won't run into them or see why to use them. Keep them in the back of your mind for when you start working on some slightly more complicated projects that might need them.
 
+### Enums
+
+Enums (short for "enumerations") map symbolic names to integer constants. They
+make code more readable and maintainable, and they offer a performance boost
+since queries involving integers are faster than those involving strings.
```

Comment:
```suggestion
Enums (short for "enumerations") map symbolic names to integer constants. They make code more readable and maintainable, and they offer a performance boost since queries involving integers are faster than those involving strings.
```
Removing manual line breaks